xapian-letor/include/xapian-letor/ranker.h

```diff
@@ -354,6 +354,67 @@ class XAPIAN_VISIBILITY_DEFAULT SVMRanker: public Ranker {
 
 };
 
+class XAPIAN_VISIBILITY_DEFAULT ListMLERanker: public Ranker {
+    /// Ranker parameters
+    std::vector<double> parameters;
+    /// Learning rate (Default is 0.001)
+    double learning_rate;
+    /// Number of iterations (Default is 10)
+    int iterations;
+
+    /** Method to train the model.
+     *
+     * @exception LetorInternalError will be thrown if training data is null.
+     */
+    void train(const std::vector<Xapian::FeatureVector> & training_data);
+
+    /** Method to save ListMLE model as db metadata.
```

Comment:
Don't document a method as "Method to save ListMLE model ...".  We know it's a method, better to just say "Save ListMLE model".

Same goes for all the other "Method to X".